### PR TITLE
Replace lorem ipsum on progress map, group detail, and event email form

### LINKED ISSUE #957: Replace lipsum on progress map and anywhere else it appears
screen shot 2015-03-26 at 10 13 36 am

### PR DESCRIPTION
**Pull request by maurizi**
Modifies progress page to change legend text on mode change, while showing
default legend text before the mode is changed.

Fixes #957

### CODE REVIEW

File: src/nyc_trees/js/src/progressPage.js [hunk 1]
```diff
@@ -24,28 +24,34 @@ var progressMap = mapModule.create({
...
-        legendEntries: '.entry',
+        legendEntries: '[data-mode]',
         actionBar: '#action-bar'
     },
     $actionBar = $(dom.actionBar);
 
 $(dom.modeChoice).click(onModeChanged);
 
-$(dom.modeChoice).first().trigger('click');
+loadLayers($(dom.modeChoice).first());
```
Comment (RickMohr) [inline src/nyc_trees/js/src/progressPage.js]: This change breaks filtering of legend entries on first load. In this picture there should only be 2 legend entries, not 4.

image

**Comment by RickMohr** [comment]
Ugly line break. Should it all be one paragraph?
image

**Comment by RickMohr** [comment]
Otherwise looks good.

**Comment by maurizi** [comment]
@RickMohr I fixed the legend entry issue.  I'm going to leave the line break thing you mentioned to the designers.

**Comment by RickMohr** [comment]
+1

**Pull request merged**